modules/ROOT/pages/queries/expressions.adoc

```diff
@@ -52,7 +52,7 @@ There may be an underscore between the `0x` or `0o` and the digits for hexadecim
 [[string]]
 == String
 
-* A `STRING` literal: `'Hello'`, `"World"`.
+* A string literal: `'Hello'`, `"World"`.
```

Comment:
Okay, I have thought this through whilst reading in context and think `STRING` does make sense for this case, sorry!